lib/components/viewers/amenities-panel.tsx

```diff
@@ -259,8 +259,16 @@ class AmenitiesPanel extends Component<Props, State> {
       >
         <ul className="related-items-list list-unstyled">
           {this._renderParkAndRides()}
-          {this._renderBikeRentalStations()}
-          {this._renderVehicleRentalStations()}
+          {!configCompanies ||
+            (configCompanies.find((company) =>
+              company.modes.includes('BICYCLE_RENT')
+            ) &&
+              this._renderBikeRentalStations())}
+          {!configCompanies ||
+            (configCompanies.find((company) =>
+              company.modes.includes('SCOOTER_RENT')
+            ) &&
+              this._renderVehicleRentalStations())}
```

Comment:
This formatting looks super weird but I guess the autoformatter is king

Reply:
weeping